cms/djangoapps/contentstore/views/tests/test_assets.py

```diff
@@ -87,6 +89,44 @@ def test_pdf_asset(self):
         self.assertEqual(content.content_type, 'application/pdf')
 
 
```

Comment:
nit: remove one of these blank lines (I think it's causing the quality build to fail). There should only be one blank line between methods